assembly/src/parser/lexer.rs

```diff
@@ -453,10 +453,7 @@ impl<'input> Lexer<'input> {
                         Token::QuotedString(self.slice_span(span))
                     });
                 },
-                c if c.is_ascii_alphanumeric() => {
-                    self.skip();
-                },
-                '_' | '$' | '-' | '!' | '?' | '<' | '>' | ':' | '.' => {
+                c if c.is_alphanumeric() || c.is_ascii_graphic() => {
```

Comment:
Isn't `is_alphanumeric()` a subset of `is_ascii_graphic()`? If so, we could do just `is_ascii_graphic()` - right?

Also, `is_ascii_graphic()` includes quotation marks, is that OK?

Reply:
Yes, you're correct, this is a redundant check. Here in the lexer, handling of quotation marks is done explicitly in a preceding case, so the fact that the ASCII graphic range includes double quotes doesn't present an issue (and we only allow ASCII graphic characters in quoted identifiers, in which case it is fine if a single-quote is present).